_layouts/events.html

```diff
@@ -8,7 +8,7 @@
 		<div class="event__big-title">
 			<span aria-label="page type">Events //</span>
 			<div>{{ page.title | replace: ' - ', ': ' | replace: ' and ', ' &amp; ' }}</div>
-		</div>
+		</div><!-- // event__big-title -->
```

Comment:
I like noting what's being closed. I'm unfamiliar with the `//` used in this, though—is that saying something, or can we leave it out?